mozjs/js/rust/magic_codegen/src/lib.rs

```diff
@@ -414,36 +361,29 @@ fn get_magic_struct_code(name: &quote::Ident,
                          -> MagicStructCode {
     let res = match *variant {
         VariantData::Struct(ref fields) => {
-            let mut structdef = Vec::new();
-            let mut slots = Vec::new();
+            // let mut structdef = Vec::new();
+            // let mut slots = Vec::new();
             let mut getters = Vec::new();
             let mut setters = Vec::new();
             let mut js_getters = Vec::new();
             let mut js_setters = Vec::new();
             let mut js_prop_specs = Vec::new();
-            structdef.push(quote! {
-                object: *mut jsapi::JSObject
-            });
             for (idx, field) in fields.iter().enumerate() {
                 let id = &field.ident;
                 let ty = &field.ty;
-                let (field, slot_num) = gen_structdef_and_slotnum(id, name, ty, idx);
-                let (getter, getter_name) = gen_getter(id, name, ty);
-                let (setter, setter_name) = gen_setter(id, name, ty);
+                // let (field, slot_num) = gen_structdef_and_slotnum(id, name, ty, idx);
```

Comment:
Let's delete code that is commented out and not in use; its in the git history if we ever need it again.

Here and above.